google/cloud/storage/transfer_manager.py

```diff
@@ -844,9 +862,249 @@ def download_chunks_concurrently(
     return None
 
 
+def upload_chunks_concurrently(
+    filename,
+    blob,
+    content_type=None,
+    chunk_size=TM_DEFAULT_CHUNK_SIZE,
```

Comment:
this is technically MPU part_size? Are using chunk_size for consistency?

Reply:
It is the part size. I am using chunk_size for consistency with the similar download method.